x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/torq/torq_params.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ActionParamsProps } from '../../../../types';
+import { TorqActionParams } from '../types';
+import { JsonEditorWithMessageVariables } from '../../json_editor_with_message_variables';
+import * as i18n from './translations';
+
+const TorqParamsFields: React.FunctionComponent<ActionParamsProps<TorqActionParams>> = ({
+  actionParams,
+  editAction,
+  index,
+  messageVariables,
+  errors,
+}) => {
+  const { body } = actionParams;
+  return (
+    <JsonEditorWithMessageVariables
+      messageVariables={messageVariables}
+      paramsProperty={'body'}
+      inputTargetValue={body}
+      label={i18n.BODY_FIELD_LABEL}
+      aria-label={i18n.BODY_FIELD_ARIA_LABEL}
+      errors={errors.body as string[]}
+      onDocumentsChange={(json: string) => {
+        editAction('body', json, index);
+      }}
+      onBlur={() => {
+        if (!body) {
+          editAction('body', '', index);
+        }
+      }}
+    />
+  );
+};
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
why disable eslint?

Reply:
This is how other connectors export the params component.